src/common/src/system_param/mod.rs

```diff
@@ -34,7 +34,7 @@ macro_rules! for_all_undeprecated_params {
         $(, { $field:ident, $type:ty, $default:expr })*) => {
         $macro! {
             { barrier_interval_ms, u32, 1000_u32 },
-            { checkpoint_frequency, u64, 10_u64 },
```

Comment:
Too frequent checkpoint has impact on read performance, I would suggest to keep it before we have enough experiments and concrete user requirements before changing.

Reply:
Reverted checkpoint frequency change.